umbra-js/test/Umbra.test.ts

```diff
@@ -27,9 +30,10 @@ const overrides = { supportPubKey: true }; // we directly enter a pubkey in thes
 const senderIndex = 2;
 const receiverIndex = 3;
 
-describe('Umbra class', () => {
+describe.only('Umbra class', () => {
   let sender: Wallet;
   let receiver: Wallet;
+  let receivers: Wallet[] = [];
```

Comment:
Let's add comments explaining why we have two variables here. Intuitively you'd think these are redundant, so if they're not let's just document why